test/pthread/cond.c

```diff
@@ -0,0 +1,41 @@
+// Generally tests pthread_cond_wait, pthread_cond_signal,
+//  pthread_cond_init
```

Comment:
Sometimes (for no real reason - and not consistently) I indent my comments when they go past a single line.

This is fixed in my next commit.